packages/orbit-components/src/deprecated/Desktop/__tests__/index.test.tsx

```diff
@@ -8,7 +8,7 @@ describe("Desktop", () => {
     render(<Desktop>kek</Desktop>);
     expect(screen.getByText("kek")).toMatchInlineSnapshot(`
       <div
-        class="inline-block sm-mm:hidden mm-lm:hidden lm-tb:hidden tb-de:hidden"
+        class="inline-block sm:max-lm:hidden mm:max-lm:hidden lm:max-tb:hidden tb:max-de:hidden"
```

Comment:
If we had `sm-mm` why did it change to `sm:max-lm`?

Reply:
it's inside Hide, will fix